scripts/ingestion/parsers/popcon_compare.py

```diff
@@ -7,18 +7,17 @@
 import csv
 import os
 import shutil
+from scripts.ingestion.constants import *
```

Comment:
please don't use `*` imports. It's hard to know what variables are being [shadowed](https://fixpython.com/2022/08/fix-python-what-is-the-problem-with-shadowing-names-defined-in-outer-scopes-0ja9koj/)